runtime-modules/content/src/nft/mod.rs

```diff
@@ -5,69 +5,36 @@ pub use types::*;
 use crate::*;
 
 impl<T: Trait> Module<T> {
-    /// Ensure nft auction can be completed
-    pub(crate) fn ensure_auction_can_be_completed(auction: &Auction<T>) -> DispatchResult {
-        let can_be_completed = if let AuctionType::English(EnglishAuctionDetails {
-            auction_duration,
-            ..
-        }) = auction.auction_type
-        {
-            let now = <frame_system::Module<T>>::block_number();
-
-            // Check whether auction time expired.
-            (now - auction.starts_at) >= auction_duration
-        } else {
-            // Open auction can be completed at any time
-            true
-        };
-
-        ensure!(can_be_completed, Error::<T>::AuctionCannotBeCompleted);
-
-        Ok(())
-    }
-
     /// Ensure auction participant has sufficient balance to make bid
     pub(crate) fn ensure_has_sufficient_balance(
         participant: &T::AccountId,
         bid: CurrencyOf<T>,
     ) -> DispatchResult {
         ensure!(
-            T::Currency::can_reserve(participant, bid),
+            T::Currency::total_balance(participant) >= bid,
```

Comment:
My best bet, while we are using reservations, is that this entire method is not needed, instead call https://docs.rs/frame-support/3.0.0/frame_support/traits/trait.ReservableCurrency.html#tymethod.can_reserve

You should double check!

Reply:
:+1: